File: ios/Keeper/Players/Player.swift
//
//  Player.swift
//  Keeper
//
//  Created by Han Hossain on 11/7/20.
//

import Foundation

struct Player {
    let firstName: String
    let lastName: String
    let position: String
    let team: String?
    
    var name: String {
        return "\(firstName) \(lastName)"
    }
}

File: src/Keeper.Core/Models/Player.cs
using System;
namespace Keeper.Core.Models
{
    public class Player
    {
        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string Position { get; set; }

        public string Team { get; set; }

        public string Name => $"{FirstName} {LastName}";

        public string PositionAndTeam => $"{Position} - {Team}";
    }
}
